dev-python/lmdb/lmdb-1.0.0.ebuild

```diff
@@ -0,0 +1,30 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+PYTHON_COMPAT=( python3_{6,7,8} )
+DISTUTILS_USE_SETUPTOOLS=bdepend
+
+inherit distutils-r1
+
+DESCRIPTION="Python bindings for the Lightning Database"
+HOMEPAGE="https://github.com/jnwatson/py-lmdb/"
+SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
+
+LICENSE="OPENLDAP"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~arm64 ~x86 ~amd64-linux ~x86-linux"
+IUSE="test"
+RESTRICT="!test? ( test )"
+
+RDEPEND="dev-db/lmdb:="
+DEPEND="${RDEPEND}"
+BDEPEND="test? ( dev-python/pytest[${PYTHON_USEDEP}] )"
+
+python_compile() {
+	LMDB_FORCE_SYSTEM=1 distutils-r1_python_compile
+}
+
+python_test() {
```

Comment:
Replace this and BDEPEND with `distutils_enable_tests pytest`

Reply:
And it adds the IUSE and RESTRICT too

Reply:
Tests aren't named `test_*` so pytest with its default options doesn't detect them. I will be adding `distutils_enable_tests pytest` on merge to remove the need for RESTRICT, IUSE and BDEPEND.